### PR TITLE
New fonts

### PR DESCRIPTION
**Pull request by sepandhaghighi**
#### Reference Issues/PRs
#155 
#### What does this implement/fix? Explain your changes.
- 10 new font added
	1. fancy131
	2. fancy132
	3. fancy133
	4. fancy134
	5. fancy135
	6. fancy136
	7. fancy137
	8. tarty1
	9. tarty2
	10. tarty3
- `mix_letters` function modified
- `CONTRIBUTING.md` modified

### CODE REVIEW

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please review this PR 💯

File: art/text_dic3.py [hunk 1]
```diff
@@ -26432,3 +26432,973 @@
...
+    'L': '𝕷',
+    'r': '𝖗',
+    'O': '𝕺',
+    '&': '&',
+    '=': '=',
+    'G': '𝕲',
+    'V': '𝖁',
+    'z': '𝖟'}
+
+tarty1_dic = {
```
Comment (sadrasabouri) [inline art/text_dic3.py:27115]: Isn't it better if we sort these lines respected to the alphabetic order (like non-ASCII fonts)?

**Review by sadrasabouri — COMMENTED** [review]
Just a tiny comment.

File: art/text_dic3.py (hunk 1 above)
Comment (sepandhaghighi) [reply art/text_dic3.py:27115]: Which lines? Dictionary's keys?

Comment (sadrasabouri) [reply art/text_dic3.py:27115]: yes

**Review by sadrasabouri — CHANGES_REQUESTED** [review]
`tarty2` seems a little bit wrong:
Screenshot from 2021-12-08 15-57-20

**Comment by sepandhaghighi** [comment]
> `tarty2` seems a little bit wrong:
> 
> Screenshot from 2021-12-08 15-57-20

It's a bit strange, let me check it.

**Comment by sepandhaghighi** [comment]
@sadrasabouri Please take a look at https://github.com/sepandhaghighi/art/pull/188/commits/5ce091b9472aaa33cd23d8988b295e7a5809e844 and https://github.com/sepandhaghighi/art/pull/188/commits/9814dde4131c629696296cf3331371a0a1ee3c99

**Review by sadrasabouri — APPROVED** [review]

**Pull request merged**